lib/Conversion/TorchToLinalg/Uncategorized.cpp

```diff
@@ -986,6 +986,24 @@ static Value createLinalgPayloadCalculationForElementwiseOp(
     auto weightedDelta = b.create<arith::MulFOp>(loc, delta, weight);
     return b.create<arith::AddFOp>(loc, start, weightedDelta);
   }
+  if (auto lerpScalar = dyn_cast<AtenLerpScalarOp>(op)) {
```

Comment:
This can be done as a decomposition. This way all three torch-mlir backend (tosa, stablehlo, linalg) benefit from this addition.